src/index.php

```diff
@@ -0,0 +1,305 @@
+<?php
+/**
+* Plugin Name: Remove Footer Links
+* Description: Remove Footer Links will try to remove site info and copyright info from your footer.
+* Version:     1.0.1
+* Runtime:     5.3+
+* Author:      Easy Blog Networks
+* Author URI:  www.easyblognetworks.com
+*/
+
+if (version_compare(PHP_VERSION, '5.3.0', '<')) {
+    ?>
+    <div id="error-page">
+        <p>This plugin requires PHP 5.3.0 or higher. Please contact your hosting provider about upgrading your
+            server software. Your PHP version is <b><?php echo PHP_VERSION; ?></b></p>
+    </div>
+    <?php
+    die();
+}
+
+// How do you debug your AJAX calls, how do you sleep at night?
+function d($s, $d = 0) {
+    $f = fopen('/tmp/yolo.txt', 'a+');
+    if ($d) {
+        fwrite($f, print_r($s, 1) . "\n");
+    } else {
+        fwrite($f, $s . "\n");
+    }
+    fclose($f);
+}
+
+class FooterLinks
+{
+    private $themes_root = '';
+    private $parent_dir = '';
+    private $child_dir = '';
+    private $child_stylesheet = '';
+    private $child_functions = '';
+    private $child_name = '';
+
+    const OptionName = 'Niteoweb.FooterLinks.Themes';
+    const nonce = 'Niteoweb.FooterLinks.Nonce';
+    const CheckHook = 'Niteoweb.FooterLinks.CheckHook';
+
+    function __construct()
+    {
+        if (is_admin()) {
```

Comment:
Using current_user_can resulted in a strange error on my test setup. I'm still looking into this.
`PHP message: PHP Fatal error:  Call to undefined function wp_get_current_user() in /srv/www/cinn/wp-includes/capabilities.php on line 448`